docs/getting-started/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Getting Started
+description: This section provides a conceptual overview of Octopus Deploy, and links to documentation that guides you through your own self-hosted or cloud-hosted Octopus server.
+position: 1
+---
+
+Welcome!
+
+This section provides an overview of Octopus Deploy concepts and links to the relevant documentation, which explore the concepts further and guides you through implementing them with your own [self-hosted](#self-hosted-octopus) or [cloud-hosted](#octopus-cloud) Octopus server.
+
+
+## How Octopus Works
+
+Octopus Deploy allows users to consistently and reliably deploy by creating a Release and deploying it following the steps defined in a Projects Deployment Process.
+
+As an Octopus user, you define the process for deploying your software. You specify the environments the applications are deployed to and who on your team can deploy to which environments. For instance, you might want QA to deploy to test environments, but not to production. Taking this approach means that even if different members of the team trigger deployments, the deployment process remains consistent. Once releases have been created, they can be deployed as many times as you need.
+
+Below outlines the main steps for doing your first deployment and some of the terms you will need to know.
```

Comment:
I'm not sure about this sentence. "Below outlines" doesn't sound right to me. You could try: 
```suggestion
The steps below walk you through your first deployment and some of the terms you will need to know.
```